example/data/input/pipeline/datamappingpipeline.json

```diff
@@ -0,0 +1,320 @@
+        "type":"INT",
+        "required":true,
+        "position":null,
+        "limitToAllowedValues":false,
+        "defaultValueExpression":null,
+        "allowedValues":[ ],
+        "precision":null,
+        "scale":null,
+        "minimum":null,
+        "maximum":null,
+        "pattern":null
+      },
+      {
+        "name":"Agent ID",
+        "description":null,
+        "type":"INT",
+        "required":true,
+        "position":null,
+        "limitToAllowedValues":false,
+        "defaultValueExpression":null,
+        "allowedValues":[ ],
+        "precision":null,
+        "scale":null,
+        "minimum":null,
+        "maximum":null,
+        "pattern":null
+      },
+      {
+        "name":"Caller Number",
+        "description":null,
+        "type":"STRING",
+        "required":true,
+        "position":null,
+        "limitToAllowedValues":false,
+        "defaultValueExpression":null,
+        "allowedValues":[ ],
+        "minimumLength":9,
+        "maximumLength":null,
+        "allowBlank":false,
+        "pattern":null
+      },
+      {
+        "name":"Call Start Time",
+        "description":null,
+        "type":"TIME",
+        "required":true,
+        "position":null,
+        "limitToAllowedValues":false,
+        "defaultValueExpression":null,
+        "allowedValues":[ ],
+        "pattern":null,
+        "minimum":null,
+        "minimumClass":null,
+        "maximum":null,
+        "maximumClass":null
+      },
+      {
+        "name":"Call End Time",
+        "description":null,
+        "type":"TIME",
+        "required":true,
+        "position":null,
+        "limitToAllowedValues":false,
+        "defaultValueExpression":null,
+        "allowedValues":[ ],
+        "pattern":null,
+        "minimum":null,
+        "minimumClass":null,
+        "maximum":null,
+        "maximumClass":null
+      },
+      {
+        "name":"Disposition",
+        "description":null,
+        "type":"STRING",
+        "required":true,
+        "position":null,
+        "limitToAllowedValues":false,
+        "defaultValueExpression":"'UNKNOWN'",
+        "allowedValues":[ ],
+        "minimumLength":null,
+        "maximumLength":null,
+        "allowBlank":false,
+        "pattern":null
+      }
+    ],
+    "validations":[
+
+    ]
+  },
+  "output":{
+    "__class__":"com.northconcepts.datapipeline.foundations.pipeline.output.ExcelPipelineOutput",
+    "providerType":"POI_XSSF",
+    "sheetName":null,
+    "sheetIndex":-1,
+    "firstRowIndex":0,
+    "firstColumnIndex":0,
+    "autofitColumns":false,
+    "autoFilterColumns":false,
+    "fieldNamesInFirstRow":true,
+    "fileSink":{
+      "__class__":"com.northconcepts.datapipeline.foundations.file.LocalFileSink",
+      "name":null,
+      "path":"data/output/test.xlsx",
+      "append":false
+    }
+  },
+  "multithreaded":true,
+  "dataMapping":{
+    "__class__":"com.northconcepts.datapipeline.foundations.datamapping.DataMapping",
```

Comment:
Can you also exclude `__class__` from DataMapping in DPF?

Reply:
done